tests/fortran/test_fortran_client.py

```diff
@@ -25,23 +25,25 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import pytest
-from os import path as osp
+import os
```

Comment:
Not sure why you've made this change. This driver now diverges from the other driver scripts.  Suggest you either revert it or update the other scripts.

Reply:
Mainly since now we're using both the `os.environ` and `os.path`. That being said, I agree with you that we should unify the convention.

Reply:
Reverted to `osp`